src/Storefront/Resources/app/storefront/src/plugin/captcha/basic-captcha.plugin.js

```diff
@@ -27,12 +27,30 @@ export default class BasicCaptchaPlugin extends Plugin {
 
         this.formPluginInstances = window.PluginManager.getPluginInstancesFromElement(this._form);
 
+        this.createFakeInput();
+
         /** @deprecated tag:v6.8.0 - HttpClient is deprecated. Use native fetch API instead. */
         this._httpClient = new HttpClient();
         this._onLoadBasicCaptcha();
         this._registerEvents();
     }
 
+    /**
+     * Creates a fake input which is required.
+     * This ensures that the form stays invalid until the captcha is solved correctly.
+     * It helps to create compatibility with other plugins that rely on the native `checkValidity()` method.
+     */
+    createFakeInput() {
+        this.fakeInput = document.createElement('input');
+        this.fakeInput.type = 'text';
+        this.fakeInput.id = 'shopware_basic_captcha_check';
+        this.fakeInput.name = 'shopware_basic_captcha_check';
+        this.fakeInput.required = true;
+        this.fakeInput.classList.add('visually-hidden');
```

Comment:
What about a11y? Won't the screen reader read this input to me and would I also be able to "tab" to it?

Reply:
Valid point. I imprved the fake input.